src/backend/PoetrySlam/schemas.py

```diff
@@ -14,3 +15,23 @@ class User(BaseModel):
 
     class Config:
         orm_mode = True
+
+
+class UserProfile(BaseModel):
+	id: UUID
+	user_name: str
+	full_name: str
+	location: str
+	years_experience: str
+	
+
+
+class PoemEntries(BaseModel):
```

Comment:
see above. I'd drop changes to the backend, does you commit message even reflect changes to this scope?